include/quicly/frame.h

```diff
@@ -252,7 +252,7 @@ inline uint64_t quicly_decodev(const uint8_t **src, const uint8_t *end)
 
     /* multi-byte */
     size_t len = 1 << (**src >> 6);
-    if (end - *src < len)
+    if (end - *src < 0 || (size_t)(end - *src) < len)
```

Comment:
I do not think we need this. `end - *src >= 0` is a precondition of calling the function.

Reply:
So we just need to convert type
Otherwise g++ will report a warning

Reply:
Adding type conversions (i.e. `(size_t)`) makes sense. Would you mind filing a separate PR that does that?